_docs/advanced-features.md

```diff
@@ -115,23 +115,25 @@ The latter is `~/.dockstore/cache/` by default and can be set to any directory l
 
 By default, Dockstore will attempt to download files up to three times. Control this with the `file-provision-retries` parameter inside `~/.dockstore/config`.
 
-## Running CWL-runner with extra tags
+## Running cwltool with extra tags
 
-When running a CWL tool, you may want to add additional parameters/flags to the `cwl-runner` command. You can do this by updating your dockstore config file (`~/.dockstore/config`).
+When running a CWL tool or workflow, you may want to add additional parameters/flags to the `cwltool` command. You can do this by updating your dockstore config file (`~/.dockstore/config`).
```

Comment:
Looking at [the code](https://github.com/dockstore/dockstore/blob/15621271f31ce4a9dd26a219eea0dea3bda9c55a/dockstore-client/src/main/java/io/dockstore/client/cli/nested/CwltoolLauncher.java#L52), it looks like the extra flags are passed to all of the implementations, not just cwltool. It would be good if Gary and/or Andrew confirmed.

Given that I think the description needs to be more generic.

Reply:
Ya, that was my initial guess.  But I wasn't entirely sure.